docs/wxm-devices/D1-bundle/wb1200-introduction.mdx

```diff
@@ -1,6 +1,6 @@
 ---
 id: D1-bundle-introduction
-title: WeatherXM D1 Bundle (WB1200) Introduction
+title: WeatherXM D1 Bundle Introduction
 hide_title: true
 sidebar_label: WeatherXM D1 Bundle-Introduction
```

Comment:
The problem is here as mentioned [before](https://github.com/WeatherXM/docs/pull/104#discussion_r1617878391). Remove the "-Introduction".